src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -473,29 +479,29 @@ private void retryCacheableRequest()
     private boolean storeRequest(
         final ListFW<HttpHeaderFW> headers)
     {
-        this.requestSlot = factory.requestBufferPool.acquire(acceptInitialId);
-        if (requestSlot == NO_SLOT)
+        this.requestSlot.value = factory.requestBufferPool.acquire(acceptInitialId);
+        if (requestSlot.value == NO_SLOT)
```

Comment:
Recommend you add an `assert requestSlot.value == NO_SLOT;` before the `acquire`, then assign to local variable `newRequestSlot` first, then only assign to `requestSlot.value` if `newRequestSlot != NO_SLOT`.

Reply:
Done.